marshal/fixtures/Serialization.golden

```diff
@@ -141,5 +141,29 @@
     {
       "Y": 1
     }
-  ]
-}
\ No newline at end of file
+  ],
+    "6: Marshal OrderedDict materialized": {
```

Comment:
The test fails because this is somehow misaligned. Did you rebuild the golden set using:

```
go test . -update 
```